securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -362,7 +362,7 @@ def user_token_added():
                 # Successfully verifying the code should redirect to the admin
                 # interface, and flash a message indicating success
                 flash_msg = self.driver.find_elements_by_css_selector(".flash")
-                assert "The two-factor code for user \"{user}\" was verified successfully.".format(
+                assert "The two-factor code for user \"{}\" was verified successfully.".format(
```

Comment:
This change looks unrelated to the dependency updates, but functional tests are passing so it's all good.

Reply:
It was actually required because the new version of mypy complained about it.